src/helper.cc

```diff
@@ -1241,7 +1237,7 @@ helperStatefulHandleRead(const Comm::ConnectionPointer &conn, char *, size_t len
 
 /// Handles a request when all running helpers, if any, are busy.
 static void
-Enqueue(helper * hlp, Helper::Xaction * r)
+Enqueue(Helper::Client * const hlp, Helper::Xaction * const r)
```

Comment:
Please add a TODO about making `hlp` a (`Pointer`?) reference instead of raw-pointer